db/migrate/20250526155610_add_index_good_jobs_concurrency_key_created_at.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddIndexGoodJobsConcurrencyKeyCreatedAt < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
+
+  def change
+    reversible do |dir|
+      dir.up do
+        # Ensure this incremental update migration is idempotent
+        # with monolithic install migration.
+        return if connection.index_exists? :good_jobs, %i[concurrency_key created_at]
+      end
+    end
+
+    add_index :good_jobs, %i[concurrency_key created_at], algorithm: :concurrently
+  end
```

Comment:
I would need to know the reason why this added to judge if it is necessary to add this migration.